src/main/java/io/fabric8/jenkins/openshiftsync/SecretManager.java

```diff
@@ -16,7 +17,7 @@ public class SecretManager {
     private final static Logger logger = Logger.getLogger(SecretManager.class.getName());
     private final static ConcurrentHashMap<String, String> trackedSecrets = new ConcurrentHashMap<>();
 
-    public static void insertOrUpdateCredentialFromSecret(final Secret secret) {
+    public static void insertOrUpdateCredentialFromSecret(final Secret secret) throws FormException {
```

Comment:
Where is `FormException` being thrown? Can we add a try/catch block around those locations and mitigate?